3p/iframe-messaging-client.js

```diff
@@ -74,8 +76,8 @@ export class IframeMessagingClient {
    *  @param {Object=} opt_payload The payload of message to send.
    */
   sendMessage(type, opt_payload) {
-    this.hostWindow_.postMessage/*OK*/(
-        serializeMessage(type, this.sentinel_, opt_payload), '*');
+    this.hostWindow_.postMessage/*OK*/(serializeMessage(
+        type, this.sentinel_, opt_payload, this.rtvVersion_), '*');
```

Comment:
The receiver will check the version to see if it matches its own runtime version. It should print error or even stop processing the message for a mismatch.